cloud/aws/templates/aws_oidc/bin/aws_cli.py

```diff
@@ -0,0 +1,34 @@
+import subprocess
+import json
+from typing import Dict
+from typing import List
+
+from cloud.shared.bin.lib.config_loader import ConfigLoader
+
+
+class AwsCli:
+    """Wrapper class that encapsulates calls to AWS CLI."""
+
+    def __init__(self, config: ConfigLoader):
+        self.config: ConfigLoader = config
+
+    def get_secret_value(self, secret_name: str) -> str:
```

Comment:
it doesn't sit well with me to get secret value here. 
Can we instead allow users to skip entering a value by entering empty string?

Reply:
I think it makes script more bullet-proof to accidental changes. That way if value is set user doesn't accidentally overrides it as they need to go to the AWS console, look at it and then modify it. It is helpful in cases when setup is run multiple times. I know you believe that setup should be run only once, but I think we need to be prepared for cases where setup needs to be run multiple times. 

And your worry is that we read the secret and it can accidentally be exposed somewhere? I can update the code to limit exposure a bit more by having is_secret_empty() method so that secret value never leaves AwsCli class .

Reply:
yes, my worry is the secret can be exposed this way. Let's do is_secret_empty(). I understand that the person running this script has access to secrets manager and can look things up, but I'd rather not expose more than we need.